src/screens/SignIn.js

```diff
@@ -46,38 +44,57 @@ export default class SignIn extends Component<Props, State> {
     });
   };
 
-  onLogin = async () => {
-    const { email } = this.state;
-
+  googleLogin = async () => {
     try {
       await this.asyncState({ isSending: true });
+      await GoogleSignin.configure();
+      const data = await GoogleSignin.signIn();
 
-      await firebase.auth().sendSignInLinkToEmail(email, {
-        url: "https://jupiter-9b937.firebaseapp.com",
-        iOS: {
-          bundleId: "com.taxischoferes"
-        },
-        android: {
-          packageName: "com.taxischoferes",
-          installApp: true,
-          minimumVersion: "1"
-        },
-        handleCodeInApp: true
-      });
+      const credential = firebase.auth.GoogleAuthProvider.credential(data.idToken, data.accessToken)
 
-      await AsyncStorage.setItem("emailForSignIn", email);
+      await firebase.auth().signInWithCredential(credential);
 
+      await AsyncStorage.setItem("userData", data);
       await this.asyncState({ isSending: false });
+      
 
```

Comment:
Lo pusiste en esta parte?

Reply:
Aqui debe ser, una vez que te tienes el usuario.

Reply:
arriba de esa línea

Reply:
Prueba con esta forma ` await this.onAuthStateChanged(user);`.

Reply:
Estoy probando esta parte, pero no me deja instalar:
![screenshot from 2018-10-07 21-33-50](https://user-images.githubusercontent.com/3487347/46590439-e1530200-ca78-11e8-8676-d3dd1fbdf587.png)

Reply:
Mi error, me faltó `yarn install`. Ahora funciona.

Reply:
lo unico que hice fue instalar el react native google signin, y con las configuraciones que estan en el primer commit

Reply:
Si, era cosa de yarn.

Reply:
Pacere que ya lo pude hacer funcionar.